### DISCUSSION TITLE
[Help Wanted] Now that the Action Bar has been deprecated, how do I generate a timestamp?

### DISCUSSION THREAD

**Discussion by umbellipher** [standardnotes/forum#2661] [General]
Nothing to add to the title above, really.

**Comment by effieeee** [comment]
Hi @umbellipher, unfortunately, there is no app native feature to replace this functionality just yet, though if you perhaps only need the timestamp when starting a new note, you can have each note use the **current date and time** as the default title. You can also customize the date and time format to your liking, as shown below:
![image](https://user-images.githubusercontent.com/54989748/207642764-daaf39c9-f2e8-4189-bb24-2233236da380.png)
